jobs/mysql_restore_job.go

```diff
@@ -341,7 +342,17 @@ func isPathExists(path string) (bool, error) {
 	}
 }
 
+func getPermissions(path string) (os.FileMode, error) {
+	info, err := os.Stat(path)
+	if err != nil {
+		return 0, err
+	}
+	return info.Mode(), nil
+}
+
 func restoreBackup(ctx context.Context, backupDirectory, mySQLDirectory string) error {
+	mysqlDirPermissions := os.FileMode(0751)
```

Comment:
Where did you get this permissions? If from some MySQL docs then let's add link to the comments.

Reply:
I copied it from default mysql folder. It is used only if this folder is absent (sth gone wrong with DB). Such a way we provide an access to DB and all the permissions should be rechecked by DBA who is responsible for this DB. By default, during the restore, permissions are set to 0750 which may cause users cannot connect.

Reply:
After initial installation, main mysql directory has permissions 0751, and so I can use shell `mysql` command to connect to the DB under non-root user. After restoring, if we don't change permissions, `xtrabackup` utility sets permissions to 0750, therefore the DB becomes inaccessible for non-root users.

As per documentation you provided there are no recommendations for the main folder. So we can set +x to allow to see the  content of the dir. What do you think?

Reply:
I didn't get an exact answer, so I changed permissions to 0750 and left a TODO in order we can address this problem when we start close work on the backup service.